src/Components/ArticleCard.css

```diff
@@ -1,12 +1,16 @@
 .articlecard-main__container {
     height: 700px;
     width: 100%;
+    background-color: gray;
     flex-direction: row;
     justify-content: center;
     align-items: center;
     border: green 2px solid;
+    transition: transform 0.2s ease-in-out;
+}
 
-
+.articlecard-main__container:hover {
+  transform: scale(1.1);
```

Comment:
Take care with this Kyle - it is leading to some undesirable behaviour and text is being pushed off screen